test/functional/page_objects/console_page.ts

```diff
@@ -286,6 +286,18 @@ export class ConsolePageObject extends FtrService {
     await this.testSubjects.click('settings-save-button');
   }
 
+  public async toggleKeyboardShortcuts(enabled: boolean) {
+    await this.openSettings();
+
+    // while the settings form opens/loads this may fail, so retry for a while
+    await this.retry.try(async () => {
+      const toggle = await this.testSubjects.find('enableKeyboardShortcuts');
+      await toggle.click();
+    });
+
+    await this.testSubjects.click('settings-save-button');
```

Comment:
Is there any thing that can be asserted upon in the ui to ensure this click did what it says it should do?

Reply:
We usually treat these helpers as test actions, so the actual assertions on whatever effect it had for the UI happens in the test itself. In the test I added above I test that it can be disabled, and that the keyboard shortcuts dont work given that we already have another test that tests the all shortcuts

Reply:
Ok, I'll approve, but :fyi: clicks w/o assertions have a tendency to become flaky (there's a little dogma in this lol).  
But, if you're good with it, I'll approve.